GetVisibleServicesByScope/function.json

```
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "v1/services/scope",
```

I think this should mirror the actual cdn link:

https://assets.cdn.io.italia.it/services/servicesByScope.json
```suggestion
      "route": "services/servicesByScope.json",
```